demos/common/cpp/models/src/associative_embedding_decoder.cpp

```diff
@@ -112,7 +112,7 @@ std::vector<Pose> matchByTag(std::vector<std::vector<Peak>>& allPeaks,
 
         if (numAdded > numGrouped) {
             cv::copyMakeBorder(matchingCost, matchingCost, 0, 0, 0, numAdded - numGrouped,
-                               cv::BORDER_CONSTANT, 10000000000);
+                               cv::BORDER_CONSTANT, 10000000);
```

Comment:
What was the problem here?

Reply:
I haven't identified exactly where the problem is, but I found some examples, when function `KuhnMunkres` doesn't work correctly for the this reason. My assumption that reason in the types - overflow or loss data. I set inf as 10^7 - so algo is correct, but I think better to set location of problem. So we can find cuter decision. I can consider this question.

Reply:
Ok. I don't thing it is that important, but I won't discourage you from investigation if you want to conduct it :)